Makefile.core.mk

```diff
@@ -0,0 +1,36 @@
+BASE := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+GOPATH = $(shell cd ${BASE}/../../..; pwd)
+TOP ?= $(GOPATH)
+
+${GOPATH}/src/istio.io/istio:
+	mkdir -p $GOPATH/src/istio.io
+	git clone https://github.com/istio/istio.git ${GOPATH}/src/istio.io/istio
+
+${GOPATH}/src/istio.io/installer:
+	mkdir -p $GOPATH/src/istio.io/installer
+	git clone https://github.com/istio/installer.git ${GOPATH}/src/istio.io/installer
+
+init: ${GOPATH}/src/istio.io/istio ${GOPATH}/src/istio.io/installer
+	mkdir -p ${GOPATH}/src/istio.io/istio
+
+test: init
+	$(MAKE) -C ${GOPATH}/src/istio.io/installer test $(MAKEFLAGS)
+
+check-stability:
+	./metrics/check_metrics.py
+
+lint:
+	@scripts/check_license.sh
+	@scripts/run_golangci.sh
+
+fmt:
+	@scripts/run_gofmt.sh
+
+fmtpy-checkandupdate:
+	@scripts/check_pyfmt.sh true
+
+fmtpy-checkonly:
+	@scripts/check_pyfmt.sh false
+
+include Makefile.common.mk
```

Comment:
i see this is a git mv, but seems like the wrong place to reference common.mk

In any regard, that can be handled as a followon - going to approve for now.